components/dada-lang/src/test_harness/lsp_client.rs

```diff
@@ -185,7 +185,7 @@ impl ChildSession {
 pub enum LspCommand {
     initialize {
         id: usize,
-        params: lsp_types::InitializeParams,
+        params: Box<lsp_types::InitializeParams>,
```

Comment:
Is this from a clippy warning?

Reply:
Because of uneven enum variant sizes?

Reply:
If so, very cool!

Reply:
Yep: https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant